ngraph/test/type_prop/proposal.cpp

```diff
@@ -99,12 +104,13 @@ TEST(type_prop, proposal_invalid_image_shape_size)
 {
     op::ProposalAttrs attrs;
     auto class_probs = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3, 4});
-    auto class_logits = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3, 4});
+    auto class_bbox_deltas = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3, 4});
     auto image_shape = make_shared<op::Parameter>(element::f32, Shape{5});
 
     try
     {
-        auto proposal = make_shared<op::Proposal>(class_probs, class_logits, image_shape, attrs);
+        auto proposal =
+            make_shared<op::Proposal>(class_probs, class_bbox_deltas, image_shape, attrs);
         // Should have thrown, so fail if it didn't
         FAIL() << "Invalid input tensor rank.";
```

Comment:
Can you add new tests for new operation?

Reply:
I will add tests but firstly I am trying to decide how to write the new operation. Current realization does not work.

Reply:
Done, Please review.